### PR TITLE
Add button to breadcrumbs bar to go full screen

### LINKED ISSUE #1032: Ability to go full screen

### PR DESCRIPTION
**Pull request by axelboc**
Fix #1032 

image

Once in fullscreen, the icon changes, so the user can press it again to exit fullscreen mode (but Escape works too of course).

image

### CODE REVIEW

File: packages/app/src/breadcrumbs/BreadcrumbsBar.tsx [hunk 1]
```diff
@@ -66,6 +79,7 @@ function BreadcrumbsBar(props: Props) {
           icon={FiMessageCircle}
           href="/" // replaced dynamically
           target="_blank"
+          style={{ marginLeft: '0.375rem' }}
```
Comment (axelboc) [inline packages/app/src/breadcrumbs/BreadcrumbsBar.tsx:82]: This spaces out the feedback button a little. Adding a class was a bit tricky, so I didn't bother.

Comment (loichuder) [reply packages/app/src/breadcrumbs/BreadcrumbsBar.tsx:82]: You could rearrange the buttons with a separator instead:

image

**Review by loichuder — APPROVED** [review]

File: packages/app/src/breadcrumbs/BreadcrumbsBar.tsx [hunk 2]
```diff
@@ -40,7 +53,7 @@ function BreadcrumbsBar(props: Props) {
         onToggle={onToggleExplorer}
       />
 
-      <Separator />
+      <Separator style={{ marginLeft: '0.375rem', marginRight: '0.875rem' }} />
```
Comment (axelboc) [inline packages/app/src/breadcrumbs/BreadcrumbsBar.tsx:56]: I now forward HTML attributes to the `Separator`'s `span` element.

**Pull request merged**